docs/app/pages/components/components-sections/wizard/marquee-wizard/marquee-wizard.component.html

```diff
@@ -262,6 +262,10 @@ <h1 class="m-t-nil">Sixth step title</h1>
         If set to
```

Comment:
@donna-marijne should this change be moved to the subclass (WizardComponent) to make this option available to both the wizard and the marquee wizard?

Reply:
Yes, I believe so. Good idea.

Reply:
Just to confirm do you want me to move the out of the marquee wizard and into the wizard component?

Reply:
Yes, push it up to the base class so that it's available in both components.
```ts
export class MarqueeWizardComponent extends WizardComponent implements OnDestroy {
```
Update documentation and tests for both.

Reply:
No problem